modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/item/provider/CPDefinitionInfoItemFieldValuesProvider.java

```diff
@@ -467,20 +467,22 @@ private List<InfoFieldValue<Object>> _getCPDefinitionInfoFieldValues(
 					cpDefinition.getStatusDate()));
 			cpDefinitionInfoFieldValues.add(
 				new InfoFieldValue<>(
-					CPDefinitionInfoItemFields.subscriptionEnabledInfoField,
+					CPDefinitionInfoItemFields.
+						paymentSubscriptionEnabledInfoField,
```

Comment:
@elocamp , These need to be ordered as well. Please fix.

Reply:
you want these fields to come after nameInfoField and pendingInfoField since you renamed them.

Reply:
Hey, @brianikim 
Is it right now?